### PR TITLE
add Flukso integration documentation

### PR DESCRIPTION
**Pull request by bertouttier**
## Proposed change

Add documentation for the new Flukso integration.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [x] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [x] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/56450
- Link to parent pull request in the Brands repository: https://github.com/home-assistant/brands/pull/2832
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/flukso.markdown [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+---
+
+This integration allows you to add [Flukso](https://flukso.net/) energy monitoring devices over the local MQTT link.
+
+This page describes how to integrate your Flukso energy monitoring devices with Home Assistant. Installation instructions for your Flukso device itself can be found on the [official website](https://flukso.net/installation).
+
+## Requirements
+
+- MQTT broker and the [MQTT integration](/integrations/mqtt/) set up in Home Assistant.
+- MQTT bridge set up between you Flukso MQTT broker and your Home Assistant MQTT Broker.
```
Comment (frenck) [inline source/_integrations/flukso.markdown]:
```suggestion
- MQTT bridge set up between your Flukso MQTT broker and your Home Assistant MQTT Broker.
```

File: source/_integrations/flukso.markdown [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+## Supported Features
+
+Both Flukso sensors and Kube sensors are supported. Multiple Flukso's are supported.
+
+- Sensors that are [tmpo](https://www.flukso.net/files/presentations/flukso.20140425.pdf) enabled will have the `total_increasing` state class.
```
Comment (frenck) [inline source/_integrations/flukso.markdown]: The end-user is unaware of what that even means. I think we should remove this line.

File: source/_integrations/flukso.markdown [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+- Your Flukso and it sensors will be automatically discovered when you set up the bridge connection between your HA MQTT broker and your Flukso's MQTT broker.
+- The integration will create a sensor for each Flukso/Kube sensor topic that it discovers on MQTT.
```
Comment (frenck) [inline source/_integrations/flukso.markdown]: I think we should not document these lines either. These are expected behavior, not unique features.

File: source/_integrations/flukso.markdown [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+
+{% include integrations/config_flow.md %}
+
+If using the [Mosquitto Broker add-on for Home Assistant](https://github.com/home-assistant/addons/blob/master/mosquitto/DOCS.md), you can add the following sample bridge configuration in your `customize` folder to set up a bridge connection to your Flukso MQTT broker:
```
Comment (frenck) [inline source/_integrations/flukso.markdown:36]: What if not using this add-on?

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_integrations/flukso.markdown (hunk 4 above)
Comment (bertouttier) [reply source/_integrations/flukso.markdown:36]: Then the configuration depends on the specific MQTT broker of the user. I added a line for the user to check the documentation of the specific broker of his set up.

**Comment by frenck** [comment]
Closing PR as the parent PR has been closed.

**Pull request closed without merging**